qemu/tests/migration_multi_host_with_reboot.py

```diff
@@ -0,0 +1,82 @@
+import logging
+import time
+from autotest.client.shared import error
+from autotest.client.shared import utils
+from virttest import utils_test
+from virttest import utils_misc
+
+
+@error.context_aware
+def run(test, params, env):
+    """
+    KVM multi-host migration test:
+
+    Migration execution progress is described in documentation
+    for migrate method in class MultihostMigration.
+    steps:
+        1) boot vm
+        2) do reboot vm and keep reboot vm with system_reset,
+           let guest in bios stage, before kernel loading
```

Comment:
1. in bios stage is migration_multi_host_with_reboot(do system_reset in code)
2. kernel is loaded is this "- timeout_random:".